.circleci/config.yml

```diff
@@ -3,22 +3,32 @@ version: 2
 jobs:
   build-linux:
     docker:
-      - image: tensorflow/tensorflow:custom-op
+      - image: tfencrypted/tf-big:0.1.0
```

Comment:
how is this going to work with building `manylinux` pip packages?

Reply:
Think it can be used for development in general though if you want to develop in the docker container.

Not sure I understand your question about `manylinux`. tf-big container is based on custom-op container so we should get building manylinux pip packages for free.